Task-44021 : Problem on MFA redirection

rdenarie opened the pull request:
Before this fix, in inPageNavigation, when the fetch see a 302 redirect, it follows automatically the redirect, but the url is not update and stay on the intermediate url (the one before the redirect)

This fix moves the instruction window.history.replaceState after the redirect, and set the current url to the last redirect result

boubaker commented on web/eXoResources/src/main/webapp/javascript/eXo/portal/InPageNavigation.js:
```diff
@@ -107,6 +106,7 @@
           })
             .then(resp => {
               if (resp && resp.status == 200) {
+                window.history.replaceState('', resp.url, resp.url);
```
Prior to this change, when the response code is different from 200, the page will be reloaded with new URL.
By using this change we may have a problem which is when having a response code different from 200, the current same page will be reloaded instead of the new one.

rdenarie replied:
What I can propose is : 
Configure the fetch to not follow the 302 redirect, and if we are in case of 302, then reload the page. 
WDYT @boubaker ?

rdenarie replied:
Updated

rdenarie committed:
Task-44021 : Problem on MFA redirection

Before this fix, in inPageNavigation, when the fetch see a 302 redirect, it follows automatically the redirect, but the url is not update and stay on the intermediate url (the one before the redirect)

This fix moves the instruction window.history.replaceState after the redirect, and set the current url to the last redirect result

rdenarie committed:
Merge branch 'develop' into fix/task-44021
